secure/storage/github/src/lib.rs

```diff
@@ -232,8 +279,8 @@ mod tests {
         let value = std::str::from_utf8(&b64_value).unwrap();
         assert_eq!(value, value2);
 
-        // Verify delete
-        github.delete(path).unwrap();
+        // Verify delete_file
```

Comment:
does the OWNER, REPOSITORY actually work?

Reply:
Hahaha, you can see the magic: https://github.com/davidiw/secure-store-test